dotnet/samples/Demos/VectorStoreRAG/RAGChatService.cs

```diff
@@ -99,6 +102,13 @@ private async Task ChatLoopAsync(CancellationToken cancellationToken)
             Console.Write("User > ");
             var question = Console.ReadLine();
 
+            // Exit the application if the user types "exit".
+            if (question is not null && question.Trim().Equals("exit", StringComparison.OrdinalIgnoreCase))
```

Comment:
On other demos if the user types empty/hit enter is a exit command, may consider changing this to no prompt as the exit.

"Exit" could be a valid command in users interaction 😊

Reply:
Sure, that makes sense.  Updating and adding some instructions on the console to explain the behavior to a user.